app/controllers/users_controller.rb

```diff
@@ -8,21 +8,60 @@ def show
   end
 
   def update
-    if @user.update(user_params)
-      redirect_to @user, notice: 'The user has been updated correctly.'
+    pass_params = password_params.to_h
```

Comment:
Why do these params need to be passed as a hash?

Reply:
to be able to apply any? method